### PR TITLE
Accordion design guidelines updated

### PR DESCRIPTION
**Pull request by kevinhatchoua**
@mmenestr first PF PR! Let's go over it after the holidays. Happy new years!

### CODE REVIEW

**Review by redallen — DISMISSED** [review]
LGTM, the v3 failing is due to CircleCI upgrading their Ruby image and not related to the v4 changes you've made here and can be ignored.

**Review by evwilkin — APPROVED** [review]
LGTM

**Comment by kevinhatchoua** [comment]
@redallen There seems to be some issue with the image resolution that was rendered for the annotated screen. Is this an issue with the file I exported? I am certain it was 2X.

File: packages/v4/src/content/design-guidelines/components/accordion/accordion.md [hunk 1]
```diff
@@ -3,10 +3,34 @@ id: Accordion
...
+This pattern is used to bring focus to a single row at the time. It is ideal for content that is mutually exclusive and does not need to be shown all at once. Whenever a new row is expanded, the previous one closes. Another benefit of this is real-estate management.
+
+### Use case
+Depending on the content, it can be useful to only expose users to one section at the time. This is particularly useful when working with forms. By default the first row is always open with all the others closed. Users can choose to close all rows, but only one can be opened at once.
+
+### Examples
+This variant is mostly used with navigation and sub-navigation.
+
+<img src="./img/single_expand_a.png" alt="Accordion single expand behavior" width="439"/>
+<img src="./img/single_expand_b.png" alt="Accordion single expand behavior" width="439"/>
```
Comment (redallen) [inline packages/v4/src/content/design-guidelines/components/accordion/accordion.md:21]: @kevinhatchoua Simply don't set a width on your images and they will take the full width of the content section of the page. They're resolution seems correct: https://patternfly-org-pr-2331-v4.surge.sh/v4/images/single_expand_a.1dc590b3bf3af5d251c03aaa153e5050.png

**Review by redallen — DISMISSED** [review]
Sorry to bother you, but you'll need to rebase or merge the latest upstream in order for your PR preview to update because of the CI changes I made yesterday.

**Comment by mmenestr** [comment]
Hi Kevin, looks good to me, the only thing is that the images are extremely small. I recommend setting the width to what the artboard width is in your sketch! But like Zack said, in this case since all your images are bigger than the max width of the page (880px I think) you can choose to omit the width specification in your img string and it should take on the right size!

File: packages/v4/src/content/design-guidelines/components/accordion/accordion.md (hunk 1 above)
Comment (kevinhatchoua) [reply packages/v4/src/content/design-guidelines/components/accordion/accordion.md]: All set.

**Comment by mmenestr** [comment]
Looks like the last image is still looking very small! Other than that looks good

**Comment by kevinhatchoua** [comment]
> Looks like the last image is still looking very small! Other than that looks good

All set.

File: packages/v4/src/content/design-guidelines/components/accordion/accordion.md [hunk 2]
```diff
@@ -3,10 +3,46 @@ id: Accordion
...
 
 ## Usage
+Accordions are one of many ways to handle large amounts of content. They are useful when you have a large amount of content, but limited space, with the header title giving an overview of the content that is hidden underneath it. Users can click or tap on the collapsed accordion to expand the content area and view additional details. Due to their high interaction cost, we recommend that you use them sparingly when you need an additional layer of structure to your content.
+
+## Single expand behavior
+This pattern is used to bring focus to a single row at the time. It is ideal for content that is mutually exclusive and does not need to be shown all at once. Whenever a new row is expanded, the previous one closes. Another benefit of this is real-estate management.
+
+### Use case
+Depending on the content, it can be useful to only expose users to one section at the time. This is particularly useful when working with forms. By default the first row is always open with all the others closed. Users can choose to close all rows, but only one can be opened at once.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/accordion/accordion.md:15]: Just noticed that it should say "one section at **a** time" and not at **the** time

File: packages/v4/src/content/design-guidelines/components/accordion/accordion.md [hunk 3]
```diff
@@ -3,10 +3,46 @@ id: Accordion
...
+
+### Examples
+This variant is mostly used with navigation and sub-navigation.
+
+<img src="./img/single_expand_a.png" alt="Accordion single expand behavior"/>
+<img src="./img/single_expand_b.png" alt="Accordion single expand behavior"/>
+
+1. **Header bar:** This interactive link expands or collapses to reveal the nested content.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/accordion/accordion.md:23]: I'm sorry I didn't catch this earlier, I promise this is the last thing you'll need to edit, but I just remembered that Abi once recommended to make these sentences in the numbered bullets start the same way in terms of their structure, so i'd reword this as follows:

1. Header: interactive link that expands or collapses to reveal the nested content.
2. Label: summary descriptions  of the nested content.
3. Content panel: area that holds the information associated with the header.

**Comment by mmenestr** [comment]
@kevinhatchoua so sorry I didn't catch these earlier but I just noticed 2 more tweaks that need to be made as commented above! I promise after this it's good to go haha

**Comment by kevinhatchoua** [comment]
@mmenestr Good catch and no worries! I appreciate the attention to details 👍🏿

**Comment by mmenestr** [comment]
@kevinhatchoua I left another comment I think you missed about switching "the" to "a"! Numbered list changes look good though!

File: packages/v4/src/content/design-guidelines/components/accordion/accordion.md (hunk 2 above)
Comment (kevinhatchoua) [reply packages/v4/src/content/design-guidelines/components/accordion/accordion.md:15]: > Just noticed that it should say "one section at **a** time" and not at **the** time

Yeah missed this comment :). As @mcarrano  said today, we'll work on improving our review process!

Comment (mmenestr) [reply packages/v4/src/content/design-guidelines/components/accordion/accordion.md]: I see that you fixed it for the sentence under "Use case", but the first sentence under "Single expand behavior" also has the same error that needs fixing!

**Review by mmenestr — APPROVED** [review]
